generate_directory_indexes.py

```diff
@@ -80,9 +80,14 @@ def render_index(prefix, order_by, contents, reverse_order, base_path):
     sorted_contents = sorted(contents, key=lambda k: k[order_by], reverse=reverse_order)
     formatted_contents = format_file_details(sorted_contents)
 
+    # If the base path has a '/' in it, assume we're working with a
+    # nested directory and update the base_path so parent directory
+    # works
+    if bool('/' in base_path):
+        base_path = '/'.join(base_path.split('/')[:-1])
```

Comment:
From my Googling of python syntax, looks like this will return everything but the last directory in the `base_path`. Why wouldn't we want only the first directory (i.e. `'/'.join(base_path.split('/')[0])`)?
For what we have now (paths with only a single `/`), it doesn't make a difference, but what if it were to have longer paths?

Reply:
Ah, good point. That's a tricky one to wrap your head around...updating!